api/src/test/java/org/pmiops/workbench/api/OfflineUserControllerTest.java

```diff
@@ -83,69 +83,73 @@ public void testBulkSyncTrainingStatusV1()
       throws org.pmiops.workbench.moodle.ApiException, NotFoundException {
     workbenchConfig.featureFlags.enableMoodleV2Api = false;
     // Mock out the service under test to simply return the passed user argument.
-    doAnswer(i -> i.getArgument(0)).when(userService).syncComplianceTrainingStatusV1(any());
+    doAnswer(i -> i.getArgument(0)).when(userService).syncComplianceTrainingStatusV1(any(), any());
```

Comment:
nit: prefer the tightest match, e.g. `any(AgentType.class)`, `anyString()`, etc. This helps avoid false positives when overloads come into play.

Reply:
I mainly learned it by repeated wrist slaps at my former employer, and I'm perpetuating the cycle of abuse here.

Seriously, though, my thoughts are
- `any()` works everywhere, so why did they bother adding `anyString()`, `anyLong()`, `any(MyClass.class)`?
- it helps with readability if I can see the type. Frequently that helps me understand the method signature a bit better when I read the test so I don't have to look it up.
- a false positive would be rare, but also very difficult to catch. I prefer verifying a tightly scoped & defined method.

Here are a couple of refs that show the strict forms but not the loose ones:
- https://www.journaldev.com/21876/mockito-argument-matchers-any-eq
- https://www.toptal.com/java/a-guide-to-everyday-mockito
- https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/ArgumentMatchers.html

Reply:
In the case of an ambiguous overload, the compiler will actually just fail. I'm not sure it's difficult to catch at all. Certainly there are scenarios where you'd need typed parameters to distinguish these different calls, which is enough of a reason for them to exist.

The documentation is maybe nice for some, but that seems subjective and comes at the price of needing to keep those types in sync. I'd say that's a wash or net negative in my mind.

The only other argument I can produce right now is that if someone comes along and adds a new method override, that might cause a compilation failure in existing tests which use `any()`.